libc/test/UnitTest/FPMatcher.h

```diff
@@ -297,31 +297,35 @@ struct ModifyMXCSR {
 #define EXPECT_FP_EXCEPTION(expected)                                          \
   do {                                                                         \
     if (math_errhandling & MATH_ERREXCEPT) {                                   \
-      EXPECT_EQ(LIBC_NAMESPACE::fputil::test_except(FE_ALL_EXCEPT) &           \
-                    ((expected) ? (expected) : FE_ALL_EXCEPT),                 \
-                (expected));                                                   \
+      EXPECT_EQ(                                                               \
+          LIBC_NAMESPACE::fputil::test_except(                                 \
+              static_cast<int>(FE_ALL_EXCEPT)) &                               \
+              ((expected) ? (expected) : static_cast<int>(FE_ALL_EXCEPT)),     \
```

Comment:
I'm not an expert but shouldn't all cast be to `unsigned int` rather then `int`?
This applies to all occurrences.

Reply:
`fetestexcept` is specified to take an `int` instead of `unsigned int` https://en.cppreference.com/w/c/numeric/fenv/fetestexcept